client/app/dedicatedCloud/datacenter/host/dedicatedCloud-datacenter-host.controller.js

```diff
@@ -51,5 +52,13 @@ angular.module("App").controller("DedicatedCloudSubDatacentersHostCtrl", ($scope
         $scope.getDatacenterCommercialRange($scope.datacenter.model.id);
     };
 
+    $scope.orderHost = (datacenter) => {
```

Comment:
Should we do this in the HTML, to declare a `a` tag for US and a button for the others ?